public/app/features/alerting/unified/utils/redux.ts

```diff
@@ -154,7 +155,14 @@ export function messageFromError(e: Error | FetchError | SerializedError): strin
       return e.statusText;
     }
   }
-  return (e as Error)?.message || String(e);
+  // message in e object, return message
+  const errorMessage = (e as Error)?.message;
+  if (errorMessage) {
+    return errorMessage;
+  }
+  // for some reason (upstream this code), sometimes we get an object without the message field neither in the e.data and nor in e.message
+  // in this case we want to avoid String(e) printing [object][object]
+  return UNKNOW_ERROR;
```

Comment:
Perhaps we could log if this happens to faro? Would it provide helpful information to us about when it happens?

Reply:
good idea!

Reply:
done